### PR TITLE
Revert "Remove unmaintained ovirt_provision plugin"

### PR DESCRIPTION
**Pull request by mmoll**
This reverts commit 3e80656d245cfd8e05fd10021189d450333a0dc6.

### CODE REVIEW

File: config/foreman.migrations/20160516143923_remove_ovirt_provision.rb [hunk 1]
```diff
@@ -1,3 +0,0 @@
-# Remove unmaintained ovirt_provision plugin
```
Comment (ekohl) [inline config/foreman.migrations/20160516143923_remove_ovirt_provision.rb:1]: I think it's safe to delete this migration, but I don't know how kafo handles missing migrations if they would have been applied.

Comment (domcleal) [inline config/foreman.migrations/20160516143923_remove_ovirt_provision.rb:1]: It's fine to remove it IIRC, but a new migration should be added to put the config/answer entries back.

Comment (mmoll) [inline config/foreman.migrations/20160516143923_remove_ovirt_provision.rb:1]: done :bowtie:

File: config/foreman.migrations/20160705103630_readd_ovirt_provision.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+answers['foreman::plugin::ovirt_provision'] ||= false
+scenario[:mapping][:'foreman::plugin::ovirt_provision'] ||= {:dir_name => 'foreman', :manifest_name => 'plugin/ovirt_provision'}
```
Comment (domcleal) [inline config/foreman.migrations/20160705103630_readd_ovirt_provision.rb]: missing `:params_name`

Comment (mmoll) [inline config/foreman.migrations/20160705103630_readd_ovirt_provision.rb]: there we go...

**Comment by domcleal** [comment]
Merged, thanks @mmoll.

Edit: Sorry, I missed that Travis CI was still running. It tested OK locally, will hopefully be fine.

**Comment by ekohl** [comment]
Looks like the push was fine, the PR failed because the remote was changed.

**Pull request merged**